src/components/common/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -211,13 +218,13 @@ export default class NameTranslation extends Mixins(CommonMixin) {
     return this.draftTranslations.filter(x => x.action !== ActionTypes.REMOVED)
   }
 
-  protected setNameTranslations (): void {
-    this.emitNameTranslations(this.draftTranslations)
```

Comment:
No need to emit to tell our parent to update the value; just set the store value here.